src/Capella.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Capella;
+
+
+/**
+ * Class Uploader
+ * @package Capella
+ *
+ * PHP SDK for CodeX Capella
+ *
+ * @example
+ * $capella = new \Capella\Uploader();
+ *
+ * $response = $capella->upload($pathToImg);
+ *
+ */
+class Uploader
+{
+
+    protected $API_URL;
+
+    public function __construct() {
+        $config = $this->loadConfig();
+
+        $this->API_URL = $config['api_url'];
+    }
+
+    /**
+     * Upload image to capella service
+     *
+     * @param string $path - path or url to image
+     * @return mixed
+     */
+    public function upload($path) {
+
+        $url = $this->API_URL;
+
+        if (file_exists($path)) {
```

Comment:
If the file does not exist (for ex. due to misprint) system decides that it's a URL. But it's not an URL.